client/signup/steps/about/index.jsx

```diff
@@ -213,10 +209,12 @@ class AboutStep extends Component {
 
 		//Site Goals
 		if ( shouldHideSiteGoals ) {
-			themeRepo = this.state.hasPrepopulatedVertical
-				? 'pub/radcliffe-2'
-				: getThemeForSiteType( siteType );
-			designType = getDesignTypeForSiteType( siteType, flowName );
+			themeRepo =
+				getSiteTypePropertyValue( 'slug', siteType, 'theme' ) || 'pub/independent-publisher-2';
+			designType =
```

Comment:
I'm a bit confused by this bit: `a ? b : c || d` is not clear, what's the operator precedence there? Can it be more explicit? It's okay to use `if` even if it's longer

Reply:
Thanks, fixed in [8ce58b6](https://github.com/Automattic/wp-calypso/pull/28672/commits/8ce58b6eea60717f8758c44edef851aaa3e13e38.)